### PR TITLE
Readme rbenv

### PR DESCRIPTION
**Pull request by jldec**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,10 +9,25 @@ To make changes, please submit a pull request. Merged changes will automatically
 - Other pages like the home page live under `_pages`
 
 ### to build and preview the site locally
-- make sure you have ruby and bundler
-- after cloning this repo, cd to this directory and run:
 
+- make sure you have ruby v2.3 or better and bundler
```
Comment (trisberg) [inline README.md:13]: any specific bundler version?

Comment (jldec) [reply README.md:13]: i'm pretty sure any recent gem install would be fine - the version check above is really more to confirm that you have bundler installed (on my recent macOS the gem install bundler fails so rbenv becomes essential)

Comment (trisberg) [reply README.md:13]: ha, looks like I already had `rbenv` installed

```
$ rbenv
rbenv 1.1.1
...
```

**Review by trisberg — APPROVED** [review]
LGTM

**Pull request merged**